src/modeler-optimisation-container/OptimEntityContainer.cpp

```diff
@@ -47,12 +47,12 @@ void OptimEntityContainer::addFromSystemComponents(
         std::vector<unsigned int> modelVariableGlobalIndices;
         modelVariableGlobalIndices.resize(variables.size());
 
+        unsigned variableGlobalIndex = 0;
```

Comment:
index is not global anymore.
Before : one index variable for all calls to addFromSystemComponent
After: index is reset to 0 for each call to addFromSystemComponents

Is it intended ?